Products/CMFPlone/controlpanel/browser/maintenance.pt

```diff
@@ -105,6 +105,10 @@
                             tal:condition="form_name"
                             tal:content="form_name">Form name</legend>
 
+                    <p>
+                        <strong i18n:translate="text_zope_database_name">Database name:</strong> <span tal:replace="view/dbName" />
```

Comment:
The `dbname` should be in an `i18n:name` for optimal translation in some languages (like right-to-left languages).